forge-gui/res/cardsfolder/upcoming/fugitive_of_the_judoon.txt

```diff
@@ -0,0 +1,10 @@
+Name:Fugitive of the Judoon
+ManaCost:4 G
+Types:Enchantment Saga
+K:Saga:3:DBToken,DBInvestigate,DBSearch
+SVar:DBToken:DB$ Token | TokenScript$ w_1_1_human_ward,w_4_4_alien_rhino | SpellDescription$ Create a 1/1 white Human creature token with ward {2} and a 4/4 white Alien Rhino creature token.
+SVar:DBInvestigate:DB$ Investigate | SpellDescription$ Investigate.
+SVar:DBSearch:AB$ ChangeZone | Cost$ Exile<1/Human> Exile<1/Artifact> | Origin$ Library | Destination$ Battlefield | ChangeType$ Card.Doctor | ChangeNum$ 1 | SpellDescription$You may exile a Human you control and an artifact you control. If you do, search your library for a Doctor card, put it onto the battlefield, then shuffle.
```

Comment:
@Northmoc 
we need to check if we can make this work without the CostDesc getting in the way

Reply:
@Simisays 
you need to move the first sentence into CostDesc, then it should look better
![image](https://github.com/Card-Forge/forge/assets/8506892/fcb6c60c-f819-4c2e-bb69-c22a0e024f85)

Reply:
A problem I've found is that you choose  to exile the human but not the artifact (meaning the effect will fizzle)

I think you need to choose them at the same time ? Is this possible currently ?

Reply:
nah that's a general problem for multiple cost parts
which might actually be a good reason to rewrite this with ChangeZone instead

Reply:
You mean two extra changezone lines (one for the artifact and the other for the human ?)

Reply:
isn't there something like EACH for that?

Reply:
I can't find an example where it's done together. I guess I'll just do it in two lines then

Reply:
no, it matters for triggers

Reply:
_Guided Passage_...

Reply:
Thanks this version seems to be working in my testing. Thoughts ?